### PR TITLE
Adding initial files for OpenShift-CI

### PR DESCRIPTION
**Pull request by dewan-ahmed**
- Used identical setup as `kam`
- used 1.13 for Go version to match the version specified on `make new-repo` script
`FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13`

### CODE REVIEW

File: openshift-ci/build-root/OWNERS [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- sbose78
+- wtam2018
+- bigkevmcd
+- chetan-rns
+- mitchharpur
+- keithchong
+- tetchel
+- amitkrout
+- jaideepr97
+- gaganhegde
```
Comment (amitkrout) [inline openshift-ci/build-root/OWNERS:19]: Place this OWNER file into the root directory. Folks listed in the owner file will get the lgtm and approve privilege from the root level.

File: openshift-ci/build-root/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+RUN yum -y install dnf httpd-tools
+
+RUN dnf -y install 'dnf-command(config-manager)'
+
+RUN dnf -y config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo
+
+RUN dnf -y install gh
```
Comment (amitkrout) [inline openshift-ci/build-root/Dockerfile:11]: For now remove line no 7 to 11. Later we will add dependencies as per the need

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: openshift-ci/build-root/OWNERS (hunk 1 above)
Comment (dewan-ahmed) [reply openshift-ci/build-root/OWNERS:19]: Done with b09222c8ba25e1c5d1cdc88a3b890335f070662a

File: openshift-ci/build-root/Dockerfile (hunk 2 above)
Comment (dewan-ahmed) [reply openshift-ci/build-root/Dockerfile]: Done with b09222c8ba25e1c5d1cdc88a3b890335f070662a

**Pull request merged**